Source/MethodCall.cs

```diff
@@ -102,6 +102,19 @@ internal partial class MethodCall : ICallbackResult, IVerifies, IThrowsResult
 		private Exception throwExceptionResponse;
 		private bool verifiable;
 
+		/// <remarks>
+		///   Only use this constructor when you know that the specified <paramref name="method"/> has no `out` parameters,
+		///   and when you want to avoid the <see cref="MatcherFactory"/>-related overhead of the other constructor overload.
+		/// </remarks>
+		public MethodCall(Mock mock, Condition condition, Expression originalExpression, MethodInfo method, IMatcher[] argumentMatchers)
```

Comment:
For the type of setups we want to install (i. e. setups that implement property getters and setters), we know in advance what kinds of matchers we'll need. The existing constructor would however force us to build an `It.IsAny<T>()` expression, which would then be put through the (slow!) `MatcherFactory.CreateMatcher` factory method. This new constructor allows us to skip these two unnecessary steps.